cmd/gp/gp.go

```diff
@@ -48,14 +48,18 @@ func Execute() {
 	}
 }
 
+func GetCmd() *cobra.Command {
+	return cmd
+}
+
 var SegContentID string
 
 func init() {
 	common.Init(cmd, conf.GP)
 
 	_ = cmd.MarkFlagRequired("config") // config is required for Greenplum WAL-G
 	// wrap the Postgres command so it can be used in the same binary
-	wrappedPgCmd := pg.Cmd
+	wrappedPgCmd := pg.GetCmd()
```

Comment:
Seems, that this breaks regular gp wal-g. For now it looks like more like an architectural problem, so lets just remove gp from universal cmd

Reply:
removed